src/main/java/com/blackducksoftware/integration/hub/sonar/compute/ComputeVulnerabilityRating.java

```diff
@@ -70,6 +71,12 @@ public void compute(final MeasureComputerContext context) {
                 rating = ComputeVulnerabilityRating.RATING_D;
             }
         }
+        if (criticalMeasure != null) {
+            shouldRate = true;
+            if (criticalMeasure.getIntValue() > 0) {
+                rating = ComputeVulnerabilityRating.RATING_E;
```

Comment:
We should probably skip straight to F for critical.

Reply:
That would make the javascript a little weird. I'd rather not skip E